### PR TITLE
general cleanup, java 11 and javadoc build errors

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -26,8 +26,8 @@
         <version>1.0.4</version>
     </parent>
 
-    <groupId>jakarta.annotation</groupId>
     <artifactId>jakarta.annotation-api</artifactId>
+    <groupId>jakarta.annotation</groupId>
```
Comment (bshannon) [inline api/pom.xml]: Why did you move the groupId after the artifactId?

File: pom.xml [hunk 2]
```diff
@@ -26,20 +26,10 @@
         <version>1.0.4</version>
     </parent>
 
-    <groupId>org.eclipse.ee4j.ca</groupId>
     <artifactId>ca-parent</artifactId>
+    <groupId>org.eclipse.ee4j.ca</groupId>
```
Comment (bshannon) [inline pom.xml]: Again, why was this moved?

File: api/pom.xml (hunk 1 above)
Comment (rotty3000) [reply api/pom.xml]: I did this so that all poms in the project were the same. I don't like when POMs get all gnarly and unorganized. They should be consistent throughout the entire project.

File: pom.xml (hunk 2 above)
Comment (rotty3000) [reply pom.xml]: see above.

File: api/pom.xml (hunk 1 above)
Comment (pzygielo) [reply api/pom.xml]: How about [POM Code Convention](http://maven.apache.org/developers/conventions/code.html#POM_Code_Convention): _specific POM convention to ordering POM elements_?

Comment (rotty3000) [reply api/pom.xml]: It's not specific ordering that I'm mostly concerned with, but rather that there is _some form_ of order. I'd be OK with following the convention.

**Comment by rotty3000** [comment]
@bshannon I've removed the whitespace changes from this PR. I'll resend it separately.
